### PR TITLE
Upgrade to ROS 2

### PR DESCRIPTION
**Pull request by PeterMitrano**
Upgrade from ROS1 to ROS2. The old ROS1 version is now on the `noetic` branch

### CODE REVIEW

File: .vscode/c_cpp_properties.json
Comment (PeterMitrano) [inline .vscode/c_cpp_properties.json:1]: Please ignore the .vscode folder in git since it has hardcoded paths for your user

File: CMakeLists.txt [hunk 1]
```diff
@@ -91,69 +30,77 @@ endif(CMAKE_SIZEOF_VOID_P MATCHES "8")
 ## Specify additional locations of header files
 ## Your package locations should be listed before other locations
 # include_directories(include)
-include_directories(${catkin_INCLUDE_DIRS} ${VICON_SDK_INCLUDE_DIR})
+include_directories(include ${Boost_INCLUDE_DIRS} ${VICON_SDK_INCLUDE_DIR})
```
Comment (PeterMitrano) [inline CMakeLists.txt:33]: I don't think we're actually using Boost -- try removing this and the `Boost_LIBRARIES` one too

File: launch/vicon_bridge.launch [hunk 2]
```diff
@@ -12,42 +13,30 @@
...
-        <!-- Transform from the mocap calibration plate to a floor aligned frame.
-             Calculated by placing 4 mocap plates to set the origin, and the positive x and y axes 
-             See issue#5 for the placement of the mocap plates  -->
-        <node pkg="tf2_ros" type="static_transform_publisher" name="fake_world_origin" required="true"
-              args="0.751024327411   -0.0822099292439   -0.842606341352        0.005632730525041  -0.016077801473191   0.015361651701277   0.999736863531414        mocap_world world_origin">
-            <param name="use_sim_time" value="$(arg use_sim_time)"/>
-        </node>
-    </group>
+    
+    <node  pkg="tf2_ros" exec="static_transform_publisher" name="victor_torso_mocap_tf_shim_broadcaster"
```
Comment (PeterMitrano) [inline launch/vicon_bridge.launch]: These numbers seem to have changed, where do they come from? Also, let's move this to the separate launch file.

**Review by PeterMitrano — COMMENTED** [review]
@Yating05  please address comments

File: .vscode/c_cpp_properties.json
Comment (Yating05) [reply .vscode/c_cpp_properties.json:1]: already added it into .gitignore

File: launch/vicon_bridge.launch (hunk 2 above)
Comment (Yating05) [reply launch/vicon_bridge.launch]: We recalibrated the static transformation from Victor root to mocap victor torso already added  Val and Victor static transform file

File: CMakeLists.txt (hunk 1 above)
Comment (Yating05) [reply CMakeLists.txt:33]: Removing Boost will cause build failure

**Review by Yating05 — APPROVED** [review]

**Comment by PeterMitrano** [comment]
Fixes #13

**Pull request merged**